packages/api/src/makeMergedSchema/rootSchema.js

```diff
@@ -1,6 +1,8 @@
 import { gql } from 'apollo-server-lambda'
 import { GraphQLDate, GraphQLTime, GraphQLDateTime } from 'graphql-iso-date'
 
+import apiPackageJson from '../../package.json'
```

Comment:
Could you use `src/../package.json` which would future-proof this if we ever decided to move the rootSchema.